Senla/DiceViewController.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+
+class DiceViewController: UIViewController {
+
+    @IBOutlet weak var rollButton: UIButton!
+    
+    @IBOutlet weak var reRollButton: UIButton!
+    
+    @IBOutlet var numbers: [UIImageView]!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        reRollButton.isHidden=true
+
+    }
+    
+
+    @IBAction func roll(_ sender: Any) {
+        let x = Int.random(in: 0...5)
+        for i in numbers.indices{
```

Comment:
Массив это уже контейнер, не надо делать цикл по индексам
```suggestion
        for item in numbers {
```

Reply:
item это и так уже будет элемент массива, у него можно будет напрямую обратиться:
`item.isHidden = true`

Reply:
Но! если ты хочешь проехать по всем элементам и с ними что то сделать, то тебе и цикл for не нужен,
у массива есть стандартный метод:
```
numbers.forEach { item in
   item.isHidden = true
 }
```

Reply:
Затем тебе достаточно просто выбрвть рандомный элемент массива и сделать его видимым:
`numbers.randomElement()?.isHidden = true`

? - потому что метод randomElement() возвращает опционал